src/thor/costmatrix.cc

```diff
@@ -294,8 +298,6 @@ void CostMatrix::Initialize(
     const google::protobuf::RepeatedPtrField<valhalla::Location>& source_locations,
     const google::protobuf::RepeatedPtrField<valhalla::Location>& target_locations) {
 
-  targets_->reserve(max_reserved_labels_count_);
```

Comment:
removing the reservation from the robin hood hash is actually better for performance! I wonder why..